src/constants/Constants.js

```diff
@@ -7,3 +7,12 @@ export const REGISTRATION_STATUS = {
   WAITLISTED: "waitlist",
   CANCELLED: "cancelled"
 };
+export const COLOR = {
+  WHITE: '#FFFFFF',
+  FONT_COLOR: '#AEC4F4',
+  CARD_PAPER_COLOR: '#273149',
+  BACKGROUND_COLOR: '#070F21',
+  LIGHT_BACKGROUND_COLOR: 'rgb(174, 196, 244, 0.19)',
+  BIZTECH_GREEN: '#7AD040',
+  TITLE_GREEN: '#96FF50'
```

Comment:
i think we should just pick one green colour

Reply:
ok let's keep BIZTECH_GREEN